packages/plugin-messagebroker/src/rabbitmq.service.ts

```
@@ -0,0 +1,216 @@
+        const currentQueueData = this.messageHandlers.get(queueName);
+        if (currentQueueData) {
+            currentQueueData.handlers.forEach(handler => {
+                try {
+                    handler(messageContent);
+                } catch (handlerError) {
+                    this.logger.error({ error: handlerError, queueName, messageId: msg.properties.messageId }, 'Error in message handler for queue');
+                    // Optionally, decide if message should be nack'd based on handler error
+                }
+            });
+        }
+        if (!(options && options.noAck)) { // Only ack if noAck is not true
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)